src/index.js

```diff
@@ -1,60 +1,47 @@
 import React from 'react';
 import TR from 'react-test-renderer';
 
-import act from './act-compat';
-import * as queries from './queries';
-import { prettyPrint } from './pretty-print';
-import * as queryHelpers from './query-helpers';
-import { resultContainer, TestHook } from './hooks';
-import { getQueriesForElement } from './get-queries-for-element';
-import { fireEvent as rntlFireEvent, NativeEvent } from './events';
+import {
+  configure as configureNTL,
+  fireEvent as rntlFireEvent,
+  getQueriesForElement,
+  NativeEvent,
+  prettyPrint,
+  proxyUnsafeProperties,
+} from './lib';
+import act, { asyncAct } from './act-compat';
+
+configureNTL({
+  asyncWrapper: async cb => {
+    let result;
+    await asyncAct(async () => {
+      result = await cb();
+    });
+    return result;
+  },
+});
 
 function render(ui, { options = {}, wrapper: WrapperComponent } = {}) {
   const wrapUiIfNeeded = innerElement =>
     WrapperComponent ? <WrapperComponent>{innerElement}</WrapperComponent> : innerElement;
 
-  let container = {};
+  let testRenderer;
 
   act(() => {
-    container = TR.create(wrapUiIfNeeded(ui), options);
+    testRenderer = TR.create(wrapUiIfNeeded(ui), options);
   });
 
-  const baseElement = queryHelpers.removeBadProperties(container.root);
-
   return {
-    container,
-    baseElement,
-    debug: (el = baseElement) => console.log(prettyPrint(el)),
-    unmount: () => container.unmount(),
+    testRenderer,
```

Comment:
Do we have to expose the test renderer here? Isn't the container enough?

Reply:
Hmm maybe! The reason I liked it is that I thought it was pretty nice to be able to use `.toJSON()` for snapshots, but I suppose we could just recommend people use `debug()` for snapshots instead. 

Including this does probably make it too easy to test implementation details.

Reply:
long story short, yeah this does need to happen. some weird stuff happens with async tests when the testRenderer isn't exposed unfortunately